Content.Shared/Animal/UdderComponent.cs

```diff
@@ -0,0 +1,58 @@
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Animals;
+
+/// <summary>
+///     Gives the ability to produce a solution;
+///     produces endlessly if the owner does not have a HungerComponent.
+/// </summary>
+[RegisterComponent]
+[AutoGenerateComponentPause]
+public sealed partial class UdderComponent : Component
```

Comment:
Any reason it's not networked?

Reply:
The reason is that they weren't before, and I only understand half of why to do certain things, when it comes to the whole networking/shared thing.

Looking at other components on shared, I now suspect almost all components on shared should be [NetworkedComponent] ? So I add it to all the components in the PR here?

Reply:
Yep. [NetworkedComponent] makes it so that adding or removing the component on the server does the same on the client, which we want here.